proc/test_thread_rand.c

```diff
@@ -0,0 +1,108 @@
+/*
+ * Phoenix-RTOS
+ *
+ * phoenix-rtos-tests
+ *
+ * Copyright 2021 Phoenix Systems
+ * Author: Hubert Buczynski
+ *
+ * This file is part of Phoenix-RTOS.
+ *
+ * %LICENSE%
+ */
+
+#include <string.h>
+#include <stdlib.h>
+#include <unistd.h>
+#include <sys/threads.h>
+
+#include "unity_fixture.h"
+
+
+typedef struct {
+	unsigned int time;
+	unsigned int id;
+} threads_self_t;
+
+
+typedef struct {
+	unsigned int cthread;
+	unsigned int fin;
+	char stack[10][512];
```

Comment:
aligned properly by accident, please add explicit alignment

Reply:
Added explicit alignment for `stack`